tests/test_hamilton_driver.py

```diff
@@ -136,7 +136,7 @@ def test_driver_variables_exposes_original_function():
         var.name: var.originating_functions for var in dr.list_available_variables()
     }
     assert originating_functions["b"] == (tests.resources.very_simple_dag.b,)
-    assert originating_functions["a"] is None
```

Comment:
Heh this was supposed to be the contract/why I was originally hesitant to make inputs in originating functions

Reply:
yep that's a good test -- and now we're changing it :)